spec/systems/search_keywords_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'search keywords', type: :system do
+  ActiveJob::Base.queue_adapter = :test
+
+  context 'given exactly match search keyword' do
+    it 'displays searched keywords' do
+      user = Fabricate(:user)
+      Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
+
+        click_button(I18n.t('auth.login'))
+      end
+
+      expect(current_path).to eql(keywords_path)
+      expect(page).to have_selector('table')
+
+      within '#searchForm' do
+        fill_in('search', with: 'AWS')
+
+        click_button(I18n.t('keyword.search'))
+      end
+
+      within 'table' do
+        within 'tbody' do
+          expect(page).to have_selector('tr', count: 1)
+
+          tr_list = all('tr')
+
+          expect(tr_list[0]).to have_selector('td', text: 'AWS')
+        end
+      end
+    end
+  end
+
+  context 'given partially match search keyword' do
+    it 'displays searched keywords' do
+      user = Fabricate(:user)
+      Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
+
+        click_button(I18n.t('auth.login'))
+      end
+
+      expect(current_path).to eql(keywords_path)
+      expect(page).to have_selector('table')
+
+      within '#searchForm' do
+        fill_in('search', with: 'AW')
+
+        click_button(I18n.t('keyword.search'))
+      end
+
+      within 'table' do
+        within 'tbody' do
+          expect(page).to have_selector('tr', count: 1)
+
+          tr_list = all('tr')
+
+          expect(tr_list[0]).to have_selector('td', text: 'AWS')
+        end
+      end
+    end
+  end
+
+  context 'given unmatch search keyword' do
+    it 'displays searched keywords' do
+      user = Fabricate(:user)
+      Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
```

Comment:
this step repeat everywhere

Reply:
Fixed in `feature/google-scraping` branch

Reply:
Fixed in d5e0593a889d2f9b0fc3595935d42366046c3692